build/Targets/Imports.targets

```diff
@@ -245,6 +249,16 @@
     </SuggestedBindingRedirects>
   </ItemGroup>
 
+  <!-- 
+       Work around some UI delays in the new project system
+
+       https://github.com/dotnet/project-system/issues/2297
```

Comment:
You should remove this - this will mean that the dependencies node in solution explorer won't function.